rocky-raft/src/rocky/raft/dto/ServerConfig.java

```diff
@@ -0,0 +1,18 @@
+package rocky.raft.dto;
+
+public class ServerConfig extends Address {
```

Comment:
I already refactored to take care of that :p

I figured ServerConfig could extend Address because but we can revert if you like, not much difference.